src/lib/Response/ResetResponse.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Response;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Reset response to tell the app to reload the location.
+ */
+class ResetResponse extends Response implements NoRenderResponse
```

Comment:
> We may want to redirect from a tab to an entirely different part of the admin.

that's the point of this kind of response. But maybe there's a confusion because of `Location` here? `App-Location` is inspired by `Location` HTTP header from redirect responses, so `Location` here is unrelated to our Location object, it can contain any URI of the admin